core/test/audits/critical-request-chains-test.js

```diff
@@ -121,14 +121,17 @@ describe('Performance: critical-request-chains audit', () => {
     };
     const context = {computedCache: new Map()};
     return CriticalRequestChains.audit(artifacts, context).then(output => {
-      expect(output.details.longestChain.duration).toBeCloseTo(656.491);
+      expect(output.details.longestChain.duration).toBeCloseTo(660.173);
       expect(output.details.longestChain.transferSize).toEqual(2468);
       expect(output).toHaveProperty('score', 0);
     });
   });
 
   it('calculates the correct chain result for passing example (no 2.)', () => {
-    const artifacts = mockArtifacts(PASSING_CHAIN_RECORDS_2);
+    const artifacts = mockArtifacts(PASSING_CHAIN_RECORDS_2, {
+      // Precision issues from converting between ms and s create slight differences during roundtrip.
+      skipVerification: true,
```

Comment:
This was unfortunate.

<img width="518" alt="image" src="https://user-images.githubusercontent.com/4071474/186002187-cf96f423-f49e-4b6a-beed-42afdf039101.png">

Reply:
we should probably put an epsilon on those comparisons, but for a quick fix, could also just pick a different timestamp :)